app/src/pages/inside/projectSettingsPageContainer/content/elements/formattedDescription/formattedDescription.jsx

```diff
@@ -22,28 +22,30 @@ import styles from './formattedDescription.scss';
 
 const cx = classNames.bind(styles);
 
-const isAnchorElement = (tagName) => tagName === 'A';
+const isAnchorElement = (target) => !!target.closest('A');
 
-export const FormattedDescription = ({ content, event }) => {
+export const FormattedDescription = ({ content, event, hasStyle }) => {
```

Comment:
Looks like these prop handles only one case.
Could we pass className prop here to override styles?

Reply:
Done.